velox/exec/benchmarks/OrderByBenchmarkUtil.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "OrderByBenchmarkUtil.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
+
+using namespace facebook::velox;
+
+namespace {
+RowTypePtr rowWithName(std::vector<TypePtr>&& types) {
+  std::vector<std::string> names;
+  for (auto i = 0; i < types.size(); ++i) {
+    names.emplace_back(fmt::format("c{}", i));
+  }
+  return ROW(std::move(names), std::move(types));
```

Comment:
VectorMaker has provided a similar function 'rowType', can we use it?
https://github.com/facebookincubator/velox/blob/main/velox/vector/tests/utils/VectorMaker.cpp#L21-L28